lldb/source/API/SBProcess.cpp

```diff
@@ -583,6 +583,14 @@ SBError SBProcess::Continue() {
   return sb_error;
 }
 
+SBError SBProcess::ContinueInDirection(RunDirection direction) {
+  ProcessSP process_sp(GetSP());
+  if (process_sp) {
+    process_sp->SetBaseDirection(direction);
+  }
```

Comment:
```suggestion
  if (ProcessSP process_sp(GetSP()))
    process_sp->SetBaseDirection(direction);
```

Reply:
It needs to be `if (ProcessSP process_sp = GetSP())`. Done.